types/lib/Server.d.ts

```diff
@@ -1610,6 +1610,10 @@ declare class Server {
                     negatedDescription: string;
                   };
                 };
+                policyName: {
```

Comment:
I am not sure why there are so many unrelated changes in this autogenerated `d.ts` file. Especially, since I've added just one field.

Please, LMK if I have something wrong in my setup.

Reply:
It is bug in `ts`, ignore it